spec/tests/yaml/swiftnav/sbp/ssr/test_MsgSsrGriddedCorrectionBounds.yaml

```diff
@@ -68,3 +68,46 @@ tests:
       length: 45
       payload: tAAAAAMAAQEKAA8BAAoAJ+gD9AFkyJZklmQCBQoQABESExQVBgoWABcYGRob
       crc: '0x5277'
+
+  - msg:
+      fields:
+        header:
+          time:
+            tow: 180
+            wn: 3
+          num_msgs: 1
+          seq_num: 1
```

Comment:
I think the sequence number might be zero-based.

Reply:
Mmmm are you sure? I set this field to 1, since the first (original) CorrectionBounds test has 1. I look the SBP documentation and it's not clear if the `seq_num` starts from 0 or 1.